proposed/swift-interop.md

```diff
@@ -57,7 +57,7 @@ We have selected the following option for supporting the Self register in the ca
 
 1. Use specially-typed argument with a type, `SwiftSelf`, to represent which parameter should go into the self register.
 
-We will provide a `SwiftSelf` type to specify "this argument goes in the self register". Specifying the type twice in a signature would generate an `InvalidProgramException`. This would allow the `self` argument to be specified anywhere in the argument list. Alternatively, many sections of the Swift ABI documentation, as well as the Clang docs refer to this parameter as the "context" argument instead of the "self" argument, so an alternative name could be `SwiftContext`.
+We will provide a `SwiftSelf` type to specify "this argument goes in the self register". Specifying the type twice in a signature would generate an `InvalidProgramException`. This would allow the `self` argument to be specified anywhere in the argument list. Alternatively, many sections of the Swift ABI documentation, as well as the Clang docs refer to this parameter as the "context" argument instead of the "self" argument, so an alternative name could be `SwiftContext`. The `UnmanagedCallersOnly` leverages the existing proposal and treats delegates annotated with `UnmanagedCallersOnly` as Swift-like functions capable of handling context registers. In the prolog of the delegate, the self register value will be loaded into the `SwiftSelf` argument.
```

Comment:
C# overloaded the delegate keyword for function pointers. We should refer to function pointers as function pointers to avoid confusion.